HIP/hip-0000-virtual-merkle.md

```diff
@@ -0,0 +1,150 @@
+- hip: <HIP number (this is determined by the HIP editor)>
+- title: Virtual Merkle
```

Comment:
Is the right title `Virtual Merkle` or `Virtual Merkle Tree`?

Reply:
I guess a better name is "On-disk Virtual Merkle Tree"